react/components/ProductsTable.tsx

```diff
@@ -22,10 +21,22 @@ const ProductsTable: FunctionComponent<Props> = (props) => {
     refundedShippingValue,
     product,
     totalRefundAmount,
-    productsValue,
     intl: { formatMessage },
   } = props
 
+  const calculateTotalProductsValue = product.length
+    ? product.reduce((total, currentProduct) => {
+        const currentValue =
+          (currentProduct.unitPrice / 100 +
+            (parseFloat(currentProduct.tax) || 0)) *
+          currentProduct.quantity
+
+        console.log(currentValue, 'current value')
+
+        return (total += currentValue)
```

Comment:
The result is the same, but I would keep it more simple and do `total + currentValue`. The `+=` mutates the variable `total` and I think we should avoid as most as possible mutating stuff.